build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.5.7"
-  kotlin("plugin.spring") version "1.7.20"
-  kotlin("plugin.jpa") version "1.7.20"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.6.0-beta"
```

Comment:
Is it ok to go to a beta version?

Reply:
i would say do not upgrade this yet

Reply:
ok with this as the security update is with in beta as metioned by others.

Reply:
Others have done so due to it being the only one that provides a needed security patch